packages/remix-dev/config.ts

```diff
@@ -161,6 +167,11 @@ export interface RemixConfig {
    */
   devServerBroadcastDelay: number;
 
+  /**
+   * A list of absolute paths that Remix will include on it's file watcher.
+   */
+  devServerAdditionalDirectories: string[];
```

Comment:
Did you mean to duplicate this?

Reply:
It's two different types, one for the app config type which matches `remix.config.js` and another that matches the outputted internal config that's used by Remix